docs/docusaurus/docs/cloud/try_gx_cloud.md

```diff
@@ -110,7 +110,9 @@ Create a Data Asset to define the data you want GX Cloud to access within Snowfl
 
     - **Username**: Enter the username you use to access Snowflake.
 
-    - **Account identifier**: Enter your Snowflake account or locator information. The locator value must include the geographical region. For example, `us-east-1`. To locate these values see [Account Identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier).
+    - **Account identifier**: Enter your Snowflake organization and account name separated by a hyphen (`oraganizationname-accountname`) or a legacy account locator (`accountname.region`) separated by a period. The legacy account locator value must include the geographical region. For example, `us-east-1`. 
```

Comment:
Minor thing, but you put the first example after the full description of what they need to provide (account name separated by a hyphen) while the second example is provided in the middle of the description. 
```suggestion
    - **Account identifier**: Enter your Snowflake organization and account name separated by a hyphen (`oraganizationname-accountname`) or your account name and a legacy account locator separated by a period (`accountname.region`). The legacy account locator value must include the geographical region. For example, `us-east-1`. 
```